tests/parametric/test_span_links.py

```diff
@@ -158,10 +211,10 @@ def test_span_link_from_distributed_w3c_headers(self, test_agent, test_library):
         assert "s:2" in tracestateDD
         assert "t.dm:-4" in tracestateDD
 
-        assert link.get("flags") == 1 | -2147483648
-        assert len(link.get("attributes") or {}) == 0
+        # link has a sampling priority of 2, so it should be sampled
+        assert link.get("flags") == 1 | -2147483648  # Sampled and Set (31 bit according the RFC)
```

Comment:
@bwoebi Should we also test scenarios where the traceflag is:
- unsampled and set
- unsampled and not-set (I don't think this scenario is possible in python since the sampling priority or traceparent will always be used when generating a link between two spans)